src/BuildToolchains.jl

```diff
@@ -81,15 +81,16 @@ function toolchain_file(bt::CMake, p::AbstractPlatform, host_platform::AbstractP
         set(CMAKE_SYSROOT /opt/$(aatarget)/$(aatarget)/sys-root/)
         """
     end
-
+    lld_str = Sys.isapple(p) ? "ld64.lld" : "ld.lld"
+    linker_str = clang_use_lld ? "/opt/bin/$(target)/$(lld_str)" : "/opt/bin/$(target)/$(aatarget)-ld"
```

Comment:
Make a function to avoid duplication (here and in meson below) for next time we'll have to change this.  Also, this can be shortened to
```julia
linker_str = "/opt/bin/$(target)/" * (clang_use_lld ? "$(lld_str)" : "$(aatarget)-ld")
```

Reply:
By make a function, do you mean a closure, or move it out of the function altogether?

Reply:
The latter